language-support/hs/bindings/src/DA/Ledger/Valuable.hs

```diff
@@ -7,6 +7,7 @@ module DA.Ledger.Valuable(
     Valuable(..), -- types which can be converted to/from a Ledger API Value
     ) where
 
+import Data.Text.Lazy (Text)
 import DA.Ledger.Types
 
 class Valuable a where
```

Comment:
I don't like the name Valuable, since it confuses "can be converted to a value" with "has monetary value", and in our domain, we often mean the latter. How about `IsValue`?

Reply:
Yeah, I agree `Valuable` is a not-that funny pun. Couldn't think of a snappy name.
`IsValue` is still a bit vague
How about `IsDamlValue` ?

Reply:
Sure - better to be explicit than funny 😺 .